internal/server/models/provideruser.go

```diff
@@ -20,6 +23,26 @@ type ProviderUser struct {
 	AccessToken  EncryptedAtRest
 	RefreshToken EncryptedAtRest
 	ExpiresAt    time.Time
+
+	Active bool
 }
 
 func (ProviderUser) IsAModel() {}
+
+func (pu *ProviderUser) ToAPI() *api.SCIMUser {
+	return &api.SCIMUser{
+		Schemas:  []string{api.UserSchema},
+		ID:       pu.IdentityID.String(),
+		UserName: pu.Email,
+		Emails: []api.SCIMUserEmail{
+			{
+				Primary: true,
+				Value:   pu.Email,
+			},
+		},
+		Active: pu.Active,
+		Meta: api.SCIMMetadata{
+			ResourceType: "User",
+		},
+	}
```

Comment:
GivenName and FamilyName are not in here. Is that expected? Are those fields only for us to consume, and will be used in a future PR?

Reply:
I actually missed them because I wasn't setting them anywhere yet, good catch